intake/source/cache.py

```diff
@@ -8,11 +8,21 @@
 import os
 import shutil
 
+from dask.bytes.utils import infer_storage_options
 from intake.config import conf
 
 logger = logging.getLogger('intake')
 
 
+def strip_http(path):
+    storage_option = infer_storage_options(path)
+
+    protocol = storage_option['protocol']
+    if protocol in ('http', 'https'):
+        return path.replace("{}://".format(protocol), '')
```

Comment:
Hm, right... so most FSs remove the protocol, but http doesn't?
This needs a comment in the code at least.

Reply:
That's right.  I'll add a comment.